core/app/class-orbit-fox-admin.php

```diff
@@ -411,14 +411,24 @@ public function get_recommended_plugins() {
 		$plugins = [
 			'optimole-wp',
 			'feedzy-rss-feeds',
-			'wpforms-lite',
 			'translatepress-multilingual',
-			'autoptimize',
 			'wp-cloudflare-page-cache',
-			'wordpress-seo',
 			'otter-blocks',
+			'wp-maintenance-mode',
+			'multiple-pages-generator-by-porthas',
+		];
+
+		$th_plugins = [
+			'wp-landing-kit' => [
+				'banner'      => esc_url( OBFX_URL ) . '/dashboard/assets/wp-landing.jpg',
+				'name'        => 'WP Landing Kit',
+				'description' => __( 'Turn WordPress into a landing page powerhouse with Landing Kit. Map domains to pages or any other published resource.', 'themeisle-companion' ),
+				'author'      => 'ThemeIsle',
```

Comment:
```suggestion
				'author'      => 'Themeisle',
```
It should be `Themeisle` with a small `i`.